charts/super-agent/charts/super-agent-deployment/tests/configmap_superagent_config_test.yaml

```diff
@@ -113,7 +113,35 @@ tests:
               auth_config:
                 private_key_path: /etc/newrelic-super-agent/keys/from-secret.key
                 provider: local
-                token_url: https://system-identity-oauth.service.eu.newrelic.com/oauth2/token
+                token_url: https://system-identity-oauth.service.newrelic.com/oauth2/token
```

Comment:
This was changed already since they pushed for it, you will need to rebase 😕

Reply:
Rebased :)